spec/features/chefs/show_spec.rb

```diff
@@ -0,0 +1,22 @@
+require "rails_helper"
+
+RSpec.describe "the chef show page" do
+  before :each do
+    @chef_1 = Chef.create!(name: "Matty Matheson")
+    @dish_1 = @chef_1.dishes.create!(name: "ribs", description: "tasty messy ribs")
+    @ingredient_1 = @dish_1.ingredients.create!(name: "raw ribs", calories: 1000)
+    @ingredient_2 = @dish_1.ingredients.create!(name: "bbq sauce", calories: 500)
+    @ingredient_3 = Ingredient.create!(name: "ranch", calories: 500)
+    @ingredient_4 = Ingredient.create!(name: "beans", calories: 500)
+  end
+
+  it "has a link to the chef's ingredients index page" do
+    visit chef_path(@chef_1)
+
+    expect(page).to have_link("Show All Ingredients")
+
+    click_link "Show All Ingredients"
+
+    expect(current_path).to eq(chef_ingredients_path(@chef_1))
+  end
+end
```

Comment:
Nice work organizing your tests and creating robust assertions! 

Testing organization is often dictated by team norms, but this is a good example of when it can make sense to test 2 different pages in the same test file. Because the link to the ingredients page, and the presence of the ingredients on that page are very closely related, I think it's totally reasonable to include both test cases in the same file. Then, it's not just testing the chef show page cause you're leaving that page, but it's not straying very far. It's up to you and neither is wrong, just figured I would mention if you felt like it was overkill to separate them into 2 different files